src/cmd/list.go

```diff
@@ -252,7 +259,23 @@ func listOutput(images, containers []map[string]interface{}) {
 			status := container[statusKey].(string)
 			imageName := container["Image"].(string)
 
-			fmt.Fprintf(writer, "%s\t%s\t%s\t%s\t%s\n", id, nameString, created, status, imageName)
+			// Prepare color for a line based on toolbox's status
+			// By default don't colorize a line
+			isRunning := false
+			if podman.CheckVersion("2.0.0") {
+				isRunning = status == "running"
+			} else {
+				isRunning, _ = regexp.MatchString("^Up [0-9]* [a-z]* ago$", status)
+			}
+			if isRunning {
+				color = "\033[1;32m" // Light Green Color
```

Comment:
Nitpick: I know the POSIX shell implementation calls this *light green color*, but I think it's more important to mention that it's supposed to be *bold*. The first SGR parameter is 1, and it stands for bold. Whether 32 is green or light green is not such a big thing because we don't have any other shade of green, or even any other color.

Reply:
I just got back to reading this. I took the comment from the Shell implementation. To be honest, before you explained it, I didn't know 1 stands for bold.

Reply:
Yes, sorry about that. In hindsight, the comments in the POSIX shell implementation weren't ideal. :/